src/Libraries/CoreNodeModels/String.cs

```diff
@@ -59,13 +62,50 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
         }
     }
 
+    [NodeName("String from Object And Format")]
+    [NodeDescription("StringfromObjectDescription", typeof(Resources))]
+    [NodeCategory("Core.String.Actions")]
+    [NodeSearchTags("FromObjectSearchTags", typeof(Resources))]
+    [OutPortTypes("string")]
+    [IsDesignScriptCompatible]
+    public class StringFromObject : ToStringNodeBase
+    {
+        [JsonConstructor]
+        private StringFromObject(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) :
+            base("__ToStringFromObjectAndFormat", inPorts, outPorts)
+        {
+            ArgumentLacing = LacingStrategy.Disabled;
+            //TODO looks like our nodemodel json constructor base implementation needs some work
+            //I see this in a few node model nodes that use default vals.
+            if (inPorts?.Count() > 1)
+            {   
+                inPorts.ElementAt(1).DefaultValue = AstFactory.BuildStringNode("F6");
+            }
+        }
+
+        public StringFromObject() : base("__ToStringFromObjectAndFormat")
+        {
+            ArgumentLacing = LacingStrategy.Disabled;
+            InPorts.Add(new PortModel(PortType.Input, this, new PortData("object", Resources.FromObjectPortDataObjToolTip)));
+            InPorts.Add(new PortModel(PortType.Input, this,
+                    new PortData("formatSpecifier",
+                    //TODO add more info here in localized form.
+                        "format specifier for numeric values",
+                        AstFactory.BuildStringNode("F6"))));
+            OutPorts.Add(new PortModel(PortType.Output, this, new PortData("string", Resources.FromObjectPortDataResultToolTip)));
+            RegisterAllPorts();
+        }
+    }
+
     [NodeName("String from Object")]
     [NodeDescription("StringfromObjectDescription", typeof(Resources))]
     [NodeCategory("Core.String.Actions")]
     [NodeSearchTags("FromObjectSearchTags", typeof(Resources))]
     [OutPortTypes("string")]
     [IsDesignScriptCompatible]
     [AlsoKnownAs("DSCoreNodesUI.StringNodes.FromObject", "DSCoreNodesUI.FromObject")]
+    [Obsolete("this node is obsolete, please use the version of string from _ with numeric format option")]
```

Comment:
?

Reply:
will clean these messages up, I actually think since I will mark the new nodes as experimental in some way I will leave this not obsolete.